projects/ngx-stories/src/lib/ngx-stories.component.ts

```diff
@@ -65,8 +65,21 @@ export class NgxStoriesComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     this.initHammer();
+    document.addEventListener('keydown', (event: KeyboardEvent) => this.handleKeyPress(event));
```

Comment:
Thanks for the contribution,
Can we use angular's [@HostListener](https://angular.dev/api/core/HostListener?tab=usage-notes) decorator in place of this ?